### ISSUE TITLE
Extension commands not found

### ISSUE THREAD

**Issue by maikypedia** [github/vscode-codeql#1677]
**Describe the bug**
In the latest version of vscode CodeQL Extension, the extension reports that the commands are not found (tested on MacOS 12.6 M1)

**Version**
1.7.3 (current version)

**To reproduce**
Install the last version of CodeQL Extension for vscode :
![image](https://user-images.githubusercontent.com/76447395/198841325-0ee7c7c2-035a-4ae3-bb74-f8bc689e7f3d.png)

Try to run any codeql command :
![image](https://user-images.githubusercontent.com/76447395/198841500-9b50a3f1-c03c-4352-ac8e-247620e48503.png)

**Additional context**
I used the default settings.

**Comment by aeisenberg** [comment]
There must have been an error on startup of the extension. Can you go to the output view and open up the CodeQL Extension Log. Paste anything relevant here.

<img width="661" alt="Extension__CodeQL_—_codeql-action" src="https://user-images.githubusercontent.com/363559/198843134-2f0ca649-cfc9-4501-89d5-045567a2f79f.png">

**Comment by maikypedia** [comment]
I don't have any output in Extension Log but I have other error messages in :

`Log (Extension Windows)`
```
[2022-10-29 19:15:28.307] [renderer3] [error] command 'codeQLDatabases.chooseDatabaseFolder' not found: Error: command 'codeQLDatabases.chooseDatabaseFolder' not found
    at S._tryExecuteCommand (vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1714:3532)
    at S.executeCommand (vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1714:3414)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async i.open (vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:608:47397)
    at async c.open (vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:608:49173
```

`Log (Extension Host)`
```
[2022-10-29 19:06:19.330] [exthost] [info] Extension host with pid 20137 started
[2022-10-29 19:06:19.416] [exthost] [info] ExtensionService#_doActivateExtension vscode.microsoft-authentication, startup: false, activationEvent: 'onAuthenticationRequest:microsoft'
[2022-10-29 19:06:19.416] [exthost] [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: true, activationEvent: '*'
[2022-10-29 19:06:19.417] [exthost] [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
[2022-10-29 19:06:19.417] [exthost] [info] ExtensionService#_doActivateExtension vscode.ipynb, startup: true, activationEvent: '*'
[2022-10-29 19:06:19.417] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode.test-adapter-converter, startup: true, activationEvent: '*', root cause: hbenl.vscode-test-explorer
[2022-10-29 19:06:19.444] [exthost] [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
[2022-10-29 19:06:19.444] [exthost] [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
[2022-10-29 19:06:19.444] [exthost] [info] ExtensionService#_doActivateExtension hbenl.vscode-test-explorer, startup: true, activationEvent: '*'
[2022-10-29 19:06:19.482] [exthost] [info] ExtensionService#_doActivateExtension vscode.github-authentication, startup: false, activationEvent: 'onAuthenticationRequest:github'
[2022-10-29 19:06:19.493] [exthost] [info] Eager extensions activated
[2022-10-29 19:06:19.500] [exthost] [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onStartupFinished'
[2022-10-29 19:06:19.500] [exthost] [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
[2022-10-29 19:06:19.500] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode-remote.remote-containers, startup: false, activationEvent: 'onStartupFinished'
[2022-10-29 19:06:19.500] [exthost] [info] ExtensionService#_doActivateExtension PKief.material-icon-theme, startup: false, activationEvent: 'onStartupFinished'
[2022-10-29 19:06:37.913] [exthost] [info] ExtensionService#_doActivateExtension GitHub.vscode-codeql, startup: false, activationEvent: 'onCommand:codeQL.checkForUpdatesToCLI'
[2022-10-29 19:06:38.180] [exthost] [error] Activating extension GitHub.vscode-codeql failed due to an error:
[2022-10-29 19:06:38.180] [exthost] [error] Error: Cannot find module 'msw/node'
Require stack:
- /Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/mocks/mock-gh-api-server.js
- /Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/mocks/vscode-mock-gh-api-server.js
- /Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:987:15)
    at Module._load (node:internal/modules/cjs/loader:832:27)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Function.r._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:14565)
    at Function.v._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:65015)
    at Function.E._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:64383)
    at Module.require (node:internal/modules/cjs/loader:1059:19)
    at p (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:699)
    at Object.<anonymous> (/Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/mocks/mock-gh-api-server.js:6:16)
    at Module.u._compile (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:1313)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
    at Module.load (node:internal/modules/cjs/loader:1035:32)
    at Module._load (node:internal/modules/cjs/loader:876:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Function.r._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:14565)
    at Function.v._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:65015)
    at Function.E._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:64383)
    at Module.require (node:internal/modules/cjs/loader:1059:19)
    at p (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:699)
    at Object.<anonymous> (/Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/mocks/vscode-mock-gh-api-server.js:8:30)
    at Module.u._compile (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:1313)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
    at Module.load (node:internal/modules/cjs/loader:1035:32)
    at Module._load (node:internal/modules/cjs/loader:876:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Function.r._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:14565)
    at Function.v._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:65015)
    at Function.E._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:64383)
    at Module.require (node:internal/modules/cjs/loader:1059:19)
    at p (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:699)
    at Object.<anonymous> (/Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/extension.js:57:37)
    at Module.u._compile (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:1313)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
    at Module.load (node:internal/modules/cjs/loader:1035:32)
    at Module._load (node:internal/modules/cjs/loader:876:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Function.r._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:14565)
    at Function.v._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:65015)
    at Function.E._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:64383)
    at Module.require (node:internal/modules/cjs/loader:1059:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Function.r [as __$__nodeRequire] (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:5:101)
    at p._loadCommonJSModule (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:66574)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 0)
[2022-10-29 19:10:48.584] [exthost] [info] Extension host terminating: received terminate message from renderer
[2022-10-29 19:10:48.591] [exthost] [info] Extension host with pid 20137 exiting with code 0
[2022-10-29 19:10:49.441] [exthost] [info] Extension host with pid 21075 started
[2022-10-29 19:10:49.441] [exthost] [info] Skipping acquiring lock for /Users/maikpedia/Library/Application Support/Code/User/workspaceStorage/f59c383134c14b7f37f039b421a08bd1.
[2022-10-29 19:10:49.465] [exthost] [info] ExtensionService#_doActivateExtension vscode.microsoft-authentication, startup: false, activationEvent: 'onAuthenticationRequest:microsoft'
[2022-10-29 19:10:49.465] [exthost] [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: true, activationEvent: '*'
[2022-10-29 19:10:49.465] [exthost] [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
[2022-10-29 19:10:49.465] [exthost] [info] ExtensionService#_doActivateExtension vscode.ipynb, startup: true, activationEvent: '*'
[2022-10-29 19:10:49.465] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode.test-adapter-converter, startup: true, activationEvent: '*', root cause: hbenl.vscode-test-explorer
[2022-10-29 19:10:49.480] [exthost] [info] ExtensionService#_doActivateExtension vscode.npm, startup: true, activationEvent: 'workspaceContains:package.json'
[2022-10-29 19:10:49.502] [exthost] [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
[2022-10-29 19:10:49.502] [exthost] [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
[2022-10-29 19:10:49.502] [exthost] [info] ExtensionService#_doActivateExtension hbenl.vscode-test-explorer, startup: true, activationEvent: '*'
[2022-10-29 19:10:49.552] [exthost] [info] ExtensionService#_doActivateExtension vscode.github-authentication, startup: false, activationEvent: 'onAuthenticationRequest:github'
[2022-10-29 19:10:49.570] [exthost] [info] Eager extensions activated
[2022-10-29 19:10:49.577] [exthost] [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onStartupFinished'
[2022-10-29 19:10:49.577] [exthost] [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
[2022-10-29 19:10:49.577] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode-remote.remote-containers, startup: false, activationEvent: 'onStartupFinished'
[2022-10-29 19:10:49.577] [exthost] [info] ExtensionService#_doActivateExtension PKief.material-icon-theme, startup: false, activationEvent: 'onStartupFinished'
[2022-10-29 19:10:58.014] [exthost] [info] ExtensionService#_doActivateExtension GitHub.vscode-codeql, startup: false, activationEvent: 'onLanguage:ql'
[2022-10-29 19:10:58.310] [exthost] [error] Activating extension GitHub.vscode-codeql failed due to an error:
[2022-10-29 19:10:58.310] [exthost] [error] Error: Cannot find module 'msw/node'
Require stack:
- /Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/mocks/mock-gh-api-server.js
- /Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/mocks/vscode-mock-gh-api-server.js
- /Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:987:15)
    at Module._load (node:internal/modules/cjs/loader:832:27)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Function.r._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:14565)
    at Function.v._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:65015)
    at Function.E._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:64383)
    at Module.require (node:internal/modules/cjs/loader:1059:19)
    at p (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:699)
    at Object.<anonymous> (/Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/mocks/mock-gh-api-server.js:6:16)
    at Module.u._compile (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:1313)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
    at Module.load (node:internal/modules/cjs/loader:1035:32)
    at Module._load (node:internal/modules/cjs/loader:876:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Function.r._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:14565)
    at Function.v._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:65015)
    at Function.E._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:64383)
    at Module.require (node:internal/modules/cjs/loader:1059:19)
    at p (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:699)
    at Object.<anonymous> (/Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/mocks/vscode-mock-gh-api-server.js:8:30)
    at Module.u._compile (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:1313)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
    at Module.load (node:internal/modules/cjs/loader:1035:32)
    at Module._load (node:internal/modules/cjs/loader:876:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Function.r._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:14565)
    at Function.v._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:65015)
    at Function.E._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:64383)
    at Module.require (node:internal/modules/cjs/loader:1059:19)
    at p (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:699)
    at Object.<anonymous> (/Users/maikpedia/.vscode/extensions/github.vscode-codeql-1.7.3/out/extension.js:57:37)
    at Module.u._compile (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:4:1313)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
    at Module.load (node:internal/modules/cjs/loader:1035:32)
    at Module._load (node:internal/modules/cjs/loader:876:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Function.r._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:14565)
    at Function.v._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:65015)
    at Function.E._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:64383)
    at Module.require (node:internal/modules/cjs/loader:1059:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Function.r [as __$__nodeRequire] (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:5:101)
    at p._loadCommonJSModule (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:66574)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 0)
[2022-10-29 19:10:59.824] [exthost] [info] ExtensionService#_doActivateExtension vscode.configuration-editing, startup: false, activationEvent: 'onLanguage:jsonc'
[2022-10-29 19:10:59.824] [exthost] [info] ExtensionService#_doActivateExtension vscode.json-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
[2022-10-29 19:10:59.824] [exthost] [info] ExtensionService#_doActivateExtension vscode.typescript-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
```

**Comment by aeisenberg** [comment]
Thanks for letting us know. We have just released v1.7.4, which should fix your issue.

**Comment by aeisenberg** [comment]
@cxkjy can you please open a new issue for this since this is a different problem. Follow the instructions above to include any interesting stack traces from the logs.
